### PR TITLE
Included prd release notes

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**
Included the March 22 prd release notes

### CODE REVIEW

**Review by JLeeChan — APPROVED** [review]

File: docs/whats-new/production-release-notes.md [hunk 1]
```diff
@@ -2,6 +2,61 @@
...
+<summary style="font-size:20px;font-weight:bold">Tenants receive a daily notification when an invoked webhook endpoint fails.
+
+</summary>
+
+Tenants will now receive a daily notification when an invoked webhook endpoint fails. The email informs you of the webhook ID, target URL and webhook event type. You have to check the target URL that has failed and fix the endpoints.
+
+</details>
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Company field is now auto-populated based on vendor's email domain when inviting user to TechPass</summary>
```
Comment (yqyeoh) [inline docs/whats-new/production-release-notes.md]: this is only in stg & not available in Prod yet, can remove from prod release notes

File: docs/whats-new/production-release-notes.md [hunk 2]
```diff
@@ -2,6 +2,61 @@
...
 > **Tip**: Click the triangle to view more details about the change.
 
+## Production release 22 March 2023
+
+Frontend version: 1.0.0-20230315.0325 | Backend version: 1.54.0-230309.0902
+
+**Changes** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Tenants receive a daily notification when an invoked webhook endpoint fails.
```
Comment (yqyeoh) [inline docs/whats-new/production-release-notes.md]: just sth i noticed, some has fullstop some don't

File: docs/whats-new/production-release-notes.md [hunk 3]
```diff
@@ -2,6 +2,61 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Production release 22 March 2023
+
+Frontend version: 1.0.0-20230315.0325 | Backend version: 1.54.0-230309.0902
```
Comment (yqyeoh) [inline docs/whats-new/production-release-notes.md]: we released a hotfix yesterday, the latest BE version for prod is 1.54.3-230321.0659

File: docs/whats-new/production-release-notes.md (hunk 1 above)
Comment (JLeeChan) [reply docs/whats-new/production-release-notes.md]: in the staging release note was it indicated that the change will not be promoted to prod? if not we should add it to manage expectation

Comment (SenthilKumarSivakami) [reply docs/whats-new/production-release-notes.md]: removed

File: docs/whats-new/production-release-notes.md (hunk 2 above)
Comment (SenthilKumarSivakami) [reply docs/whats-new/production-release-notes.md]: fixed.

File: docs/whats-new/production-release-notes.md (hunk 3 above)
Comment (SenthilKumarSivakami) [reply docs/whats-new/production-release-notes.md]: so i should change only in the prod release note, right? that is done.

File: docs/whats-new/production-release-notes.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply docs/whats-new/production-release-notes.md]: updated the staging with the note Jerome suggested.

**Pull request merged**